src/Westsworld/TimeAgo/Translations/Id.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Westsworld\TimeAgo\Translations;
+
+use \Westsworld\TimeAgo\Language;
+
+/**
+ * Indonesian translations
+ */
+class En extends Language
```

Comment:
remember to change class name from En to Id

Reply:
Resolved! :+1: